internal/proc/status.go

```diff
@@ -18,10 +18,27 @@ import (
 	"bufio"
 	"fmt"
 	"os"
-	"os/exec"
+	"strconv"
 	"strings"
+
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/umoci/pkg/idtools"
+)
+
+var (
+	overflowUid = "65534"
+	overflowGid = "65534"
 )
 
+func init() {
```

Comment:
Could we turn this into a `sync.Once` and only run when needed?  I am no fan of `init()`s in a library as it ca accumulate to slower start-up times.

Reply:
Fair point, will switch.